pkg/downloader/chart_downloader.go

```diff
@@ -97,20 +97,29 @@ func (c *ChartDownloader) DownloadTo(ref, version, dest string) (string, *proven
 		return "", nil, err
 	}
 
-	data, err := g.Get(u.String(), c.Options...)
```

Comment:
Can we fetch the chart metadata only? And compare if the upstream content has changed?

That still incurs a network access, but avoids downloading the content if unchanged.